defaults/main.yml

```diff
@@ -58,15 +58,19 @@ traefik:
 portal_backend_image_version: "{{ hub_image_version | default('latest') }}"
 portal_image_version: latest
 reverse_proxy_image_version: 2.2
+session_management_image_version: latest
+cloudxr_management_image_version: latest
 
 mailserver_image: namshi/smtp:latest
 message_queue_image: redis:5
 database_image: postgres:9.6
 # legacy parameter to specify portal backend image is hub_image. Defaults to registry.docker.innoactive.de/portal_backend:latest
 portal_backend_image: registry.docker.innoactive.de/{{ hub_image | default('portal_backend') }}:{{ portal_backend_image_version }}
-portal_image: registry.docker.innoactive.de/hub_portal_web:{{ portal_image_version }}
+portal_image: registry.docker.innoactive.de/portal_frontend:{{ portal_image_version }}
```

Comment:
this is probably the wrong image

Reply:
why?

Reply:
isn't this the backend container var? if not than I got it confused with the `hub_web` image..

Reply:
`portal_image` is the portal frontend, `portal_backend_image` is the portal backend (f.k.a `hub_image`)